client/me/purchases/manage-purchase/index.jsx

```diff
@@ -261,16 +253,12 @@ class ManagePurchase extends Component {
 		const { purchase, translate } = this.props;
 		const { id } = purchase;
 
-		if (
-			isExpired( purchase ) ||
-			! hasPrivacyProtection( purchase ) ||
-			! this.props.selectedSite
-		) {
+		if ( isExpired( purchase ) || ! hasPrivacyProtection( purchase ) ) {
 			return null;
 		}
 
 		return (
-			<CompactCard href={ cancelPrivacyProtection( this.props.selectedSite.slug, id ) }>
+			<CompactCard href={ cancelPrivacyProtection( this.props.siteSlug, id ) }>
```

Comment:
This link necessitates modification of `CancelPrivacyProtection` to prepare it to work with no `selectedSite`. With no changes, this would redirect away from cancel privacy protection route.